cmd/ddev/cmd/scripts/test_ddev.sh

```diff
@@ -87,7 +87,7 @@ function cleanup {
 
 ddev config --project-type=php --docroot=web --disable-upload-dirs-warning || (printf "\n\nPlease run 'ddev debug test' in the root of the existing project where you're having trouble.\n\n" && exit 4)
 
-printf "RUN apt-get update\nRUN curl -I https://www.google.com\n" > .ddev/web-build/Dockerfile.test
+printf "RUN timeout 120 apt-get update || true\nRUN curl -I https://www.google.com\n" > .ddev/web-build/Dockerfile.test
```

Comment:
This does no harm, but wow, we shouldn't have to do this :)

Reply:
There is a possibility of getting stuck in `TestCmdDebugTest` because of this `apt-get update`.
So I added a timeout here.

https://github.com/ddev/ddev/actions/runs/11927128822/job/33242041249#step:16:18865